concordia/views.py

```diff
@@ -1717,6 +1717,14 @@ def serialize_object(self, obj):
             },
         }
 
+        if project.topics:
+            metadata["topics"] = {}
```

Comment:
Is this supposed to be a list of dictionaries? The code below will lose data if there's more than one topic.

Reply:
Fixed